spec/sneakers/worker_spec.rb

```diff
@@ -296,6 +308,10 @@ def work(msg)
         @dummy_q = DummyWorker.new.queue
         @dummy_q.opts[:connection].must_equal(@connection)
       end
+
+      it "should build a queue without an exchange" do
```

Comment:
I find this test to be confusing. What does a queue have to do with an exchange? If there is no way to make it clearer, let's drop it.

Reply:
You're right, after re-reading, this is really confusing. Sorry, I'm not familiar at all with the test logic. I've made a correction for this test which, maybe, makes more sense.